dnas/mewsfeed/helpers/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+use hdk::prelude::*;
```

Comment:
move to top level /crates/helpers
or break out into its own repo
or PR it to 
https://github.com/holochain/hc-utils